### DISCUSSION TITLE
Only WPGraphQL? Could also use GraphQL API for WordPress?

### DISCUSSION THREAD

**Discussion by leoloso** [wpengine/faustjs#59] [General]
Hi, I just learnt about this project today through the WP Owls newsletter.

Question: is the requirement for WPGraphQL mandatory? Or another GraphQL server for WordPress can also do?

My intention would be to power the Headless framework using the [GraphQL API for WordPress](https://github.com/GraphQLAPI/graphql-api-for-wp), which is another GraphQL server of which I'm the author. I wonder if this is doable and, if so, what requirements I should satisfy.

Thanks

**Comment by trevanhetzel** [comment]
Hey there, sorry on a late reply from us! We are currently optimizing for WPGraphQL and its schema. So for now, yes, WPGraphQL is mandatory and we aren't supporting another GraphQL server. That said, this project is still in very early stages of development, so there is definitely potential for us to consider supporting other GraphQL servers in the future.

**Comment by leoloso** [comment]
Ok, thanks for your response. I'll be watching then
